site/docs/getting-started.md

```diff
@@ -5,30 +5,38 @@ redirect_from: "docs/index.html"
+1. Install **Stylable** from our GitHub [repository](https://github.com/wix/stylable).
 
-We want to extend CSS so that it is easier to use in a component ecosystem, but without losing the aspects of CSS that make it great: declarative, familiar, static and fast. So we created **Stylable**.
+2. Install the **Stylable** [integrations package](https://github.com/wix/stylable-integration).
 
-**Styable** is a preprocessor that allows you to write style rules in CSS syntax, with some extensions that we believe adhere to the spirit of CSS.
+3. While **Stylable** can work with any type system, the most commonly used is React. Install the latest version.
 
-So what does **Stylable** do?
+## What can I do
 
-*  Scopes styles to components so they don't "leak" and clash with other styles.
-* Enables custom pseudo-classes and pseudo-elements that abstract the internal structure of a component. These can then be styled externally. For example, you can style the label inside a button, or style the play button of a video player from outside these components.
-* Uses themes so you can apply different look and feel across your web application, for example, the same component can use a Wix theme or a Bootstrap theme depending on the page.
+* **Style** - If you are looking to style a component, follow the steps in the [Stylable Basics Guide](../docs/guides/components-basics.md).
 
-At build time, the preprocessor converts the **Stylable** CSS into flat, static, valid vanilla CSS that works cross-browser.
+* **Build an app** - If you want to use **Stylable** while building a web application, go to [Build a Stylable Application](../docs/guides/stylable-application.md). Read the steps and recommendations for building a performant web application that doesn't depend on other libraries for styling.
+
+* **Create a library** - If you want guidance in creating a component library, read [Create a Stylable Component Library](../docs/guides/stylable-component-library.md). Here you can follow our recommendations for building a library that can be used cross-project and includes theming, external styling and reuable components.
+
+* **Learn** - Take a look at our reference documents to get more acquanted with our code and for specific code examples.
+
+* **Remember** - Our [cheatsheet](../docs/getting-started/cheatsheet.md) helps you keep track of **Stylable** syntax.
 
-Viva CSS, and welcome **Stylable**. We hope you like it. Learn how to use it with step by step instructions and code examples.
 
 ## Components
 
-[Stylable-components](https://github.com/wix/stylable-components) - **Stylable**, fully-tested & strictly-typed component library based on React.
+**Stylable** includes a rich [component library](https://github.com/wix/stylable-components) for out-of-the-box use.
+**Stylable** components are a fully-tested & strictly-typed component library based on React.
```

Comment:
Replace with:

We recommend using our component library, [**stylable-components**](https://github.com/wix/stylable-components) . **Stylable Components** mare a fully-tested & strictly-typed React components